packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts

```diff
@@ -0,0 +1,37 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
+import { ISparklineProps, ISparklineStyles } from './Sparkline.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+
+/**
+ * @internal
+ */
+export const sparklineClassNames: SlotClassNames<ISparklineStyles> = {
+  inlineBlock: 'fui-sprk__inlineBlock',
+  valueText: 'fui-sprk__valueText',
+};
+
+/**
+ * Base Styles
+ */
+const useStyles = makeStyles({
+  inlineBlock: {
+    display: 'inline',
+  },
+  valueText: {
+    fontSize: tokens!.fontSizeBase200,
```

Comment:
why is ! needed

Reply:
Okay here there is no need. tokens would never be null. Will remove this.